README.md

```diff
@@ -53,6 +53,7 @@ frame = {
 frame = {
     'frameType': FRAMETYPE_NOTIFICATION,
     'source':    1,
+    'movementDuration': self.tsMovementStop - self.tsMovementStart, # amount of time robot moved until it stopped 
```

Comment:
no. Your frame doesn't contain `self.tsMovementStop - self.tsMovementStart`, that's how you compute it. It contains a value. In your comment, indicate the unit is seconds.